static/docs/philosophy/related-technologies.md

```diff
@@ -81,7 +81,8 @@ process.
      want to see in your Git repository) in a local key-value store and use file
      symlinks instead of the actual files.
 
-   - DVC uses hardlinks instead of symlinks to make user experience better.
+   - DVC uses reflinks (or hardlinks) instead of symlinks to make user
```

Comment:
I think the previous paragraph should be updated. I don't see how two of them make sense together. @dmpetrov @efiop what did we want to actually say here? are you familiar with git-annex internals?

Reply:
@shcheklein Ah, right, this is an old one. Not sure why the first paragraph is that way. I would probably remove it for now.

Also, would rephrase a little: `DVC can use reflinks or hardlinks instead of symlinks to make user experience better.`

Reply:
Updated.